Gemfile.lock

```diff
@@ -68,6 +68,8 @@ GEM
       tzinfo (~> 2.0)
```

Comment:
## `README.md` File:

- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**
![Screenshot from 2023-09-08 17-32-07](https://github.com/kayjeee/Hi-Blog_app_kayjeee/assets/83042742/b6c35296-ce6e-4be2-9df6-7f621734a819)